Telemetry/Telemetry.swift

```diff
@@ -42,17 +41,17 @@ public class Telemetry {
         backgroundTasks[pingBuilderType.PingType] = UIBackgroundTaskInvalid
     }
 
-    public func hasPingType(_ pingType: String) -> Bool {
+    func hasPingType(_ pingType: String) -> Bool {
```

Comment:
I know we call these from AppEvents.swift now, but I think they should remain `public` in case an app needs fine-grained control over these events in the future.